apps/sel4test-tests/src/tests/scheduler.c

```diff
@@ -1625,7 +1625,7 @@ static int test_simple_preempt(struct env *env)
     return sel4test_get_result();
 }
 DEFINE_TEST(SCHED0021, "Test for pre-emption during running of many threads with equal prio", test_simple_preempt,
-            true);
+            !config_set(CONFIG_SIMULATION));
```

Comment:
Same here, should have a comment why this does not work in the simulation. We could mention https://github.com/seL4/sel4test/issues/42 here.